src/main/java/net/neoforged/neoforge/network/connection/ChannelAwareFriendlyByteBuf.java

```diff
@@ -1,35 +0,0 @@
-/*
- * Copyright (c) NeoForged and contributors
- * SPDX-License-Identifier: LGPL-2.1-only
- */
-
-package net.neoforged.neoforge.network.connection;
-
-import io.netty.buffer.ByteBuf;
-import io.netty.channel.ChannelHandlerContext;
-import net.minecraft.network.FriendlyByteBuf;
-import org.jetbrains.annotations.ApiStatus;
-import org.jetbrains.annotations.Nullable;
-
-@ApiStatus.Internal
-public final class ChannelAwareFriendlyByteBuf extends FriendlyByteBuf {
```

Comment:
Is there no way to the retrieve the `ChannelHandlerContext` from an FBB anymore? I have an upcoming use case for that actually. (Serializing custom ingredients as stack lists to vanilla clients by checking the connection attributes in the StreamCodec#encode).

Reply:
Fine to remove for now - will add it back if necessary.